tests/test_preview.py

```diff
@@ -216,16 +216,19 @@ def test_print_letter_returns_200(print_letter_template):
     pytest.mark.xfail(('999', 'doesnt_exist.png'), raises=BadRequest),
 ])
 def test_getting_logos(client, dvla_org_id, expected_filename):
-    assert get_logo_filename(dvla_org_id) == expected_filename
-
-
-@pytest.mark.parametrize('filename', LOGO_FILENAMES.values())
-def test_that_logo_files_exist(filename):
-    assert os.path.isfile(
-        os.path.join(
-            os.path.dirname(os.path.realpath(__file__)),
-            '..',
-            'static', 'images', 'letter-template',
-            filename
+    assert get_logo(dvla_org_id).raster == expected_filename
+
+
+@pytest.mark.parametrize('logo', LOGOS.values())
+def test_that_logo_files_exist(logo):
```

Comment:
Add a test to handle the cases where the LOGO has neither a rastor or a vector

Reply:
Tests added in d92fd96